From 1a2641f5872e3c56b5509dc86a7c64f2f760f4b5 Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Fri, 17 Jan 2025 19:31:35 +0100
Subject: [PATCH 1505/1695] clk: stm32mp21: remove define CK_SCMI_ICN_VID

ck_icn_vid is removed in STM32MP21 series, so remove the associated define

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 1fa417edf04658338643d7bb4ff582172b50dfbf clk: stm32mp21: remove define CK_SCMI_ICN_VID]
Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Change-Id: I4f8c6ecbf5aea2149a1f5eb20d88f9072da34a04
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/429886
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
---
 core/include/dt-bindings/clock/st,stm32mp21-rcc.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/core/include/dt-bindings/clock/st,stm32mp21-rcc.h b/core/include/dt-bindings/clock/st,stm32mp21-rcc.h
index 04d7d5f35..4f4dccdf1 100644
--- a/core/include/dt-bindings/clock/st,stm32mp21-rcc.h
+++ b/core/include/dt-bindings/clock/st,stm32mp21-rcc.h
@@ -322,7 +322,6 @@
 #define CK_SCMI_ICN_DISPLAY	3
 #define CK_SCMI_ICN_HSL		4
 #define CK_SCMI_ICN_NIC		5
-#define CK_SCMI_ICN_VID		6
 #define CK_SCMI_FLEXGEN_07	7
 #define CK_SCMI_FLEXGEN_08	8
 #define CK_SCMI_FLEXGEN_09	9
-- 
2.47.2

